core/src/main/scala/tofu/lift/Unlift.scala

```diff
@@ -118,11 +109,32 @@ object Unlift extends UnliftInstances1 {
 }
 
 private[lift] trait UnliftInstances1 {
-  implicit def unliftReaderT[F[_]: Applicative, R]: Unlift[F, ReaderT[F, R, *]] = {
-    type RT[a] = ReaderT[F, R, a]
-    new Unlift[F, RT] {
-      def lift[A](fa: F[A]): RT[A] = ReaderT.liftF(fa)
-      def unlift: RT[RT ~> F]      = ReaderT(r => funK[RT, F](_.run(r)).pure[F])
+  implicit def unliftWithRun[F[_], G[_], A](implicit wr: WithRun[F, G, A]): Unlift[G, F] = wr
```

Comment:
Why do we need this? WithRun extends Unlift

Reply:
I suppose that we can summon WithRun but not Unlift. Previous instance unliftReaderT was created for same purpose as it duplicates the code from WithRun instance.

Reply:
Probably, it should've been removed, when Unlift became a parent of WithRun. Please, make sure it's really needed.

Reply:
Need to reflect this in an issue at least

Reply:
It`s not much of a problem for me, since _unliftReaderT_ do the wrok, but we have same situation as in Provide and RunContext

Reply:
Is this a _real_ use case when you have `ReaderT[ReaderT[IO, R1, *], R2, *]`?
You'd better use `(R1, R2)` as an environment and optics.